assets/locales/en/translation.json

```diff
@@ -155,9 +155,9 @@
   "xCoordinate": "X: {{x}}",
   "yCoordinate": "Y: {{y}}",
   "SauceLabs Data Center": "SauceLabs Data Center",
-  "US": "US",
-  "EU": "EU",
-  "UP": "Appium Inspector will use the new Sauce Labs Unified Platform US endpoint by default. If you want to use the old US endpoint then do the following: Go to the 'Custom server'-tab, add 'ondemand.saucelabs.com' as a remote host, use port '433', the remote path is '/wd/hub' and last but not least enable 'SSL'.",
+  "US-West": "US West",
+  "US-East": "US East",
+  "EU-Central": "EU",
```

Comment:
I was thinking maybe it would make sense to have more sophisticated labels there, e.h. Central Europe, Unites States West, etc. ?

Reply:
I could use the 'official' labels from the UI: `US West 1`, `EU Central 1`, `US East 4`?

Reply:
whatever is more convenient for users to understand. It makes sense to align it with the official documentation though

Reply:
Applied UI labels `US West 1`, `EU Central 1` and `US West 4`